lib/session.js

```
@@ -125,236 +124,237 @@ function sockWrite(pdu, closeAfterSend) {
 
 		// Request DLRs!
 		if (smsOptions.dlr) {
-			pduObj.params.registered_delivery	= 0x01;
+			pduObj.params.registered_delivery = 0x01;
 		}
 
-		that.log.debug(logPrefix + 'pduObj: ' + JSON.stringify(pduObj));
+		log.debug('larvitsmpp: lib/session.js: sendLongSms() - pduObj: ' + JSON.stringify(pduObj));
 
-		that.send(pduObj, function (err, retPduObj) {
+		that.send(pduObj, function(err, retPduObj) {
```

ESLint

[error] Missing space before function parentheses.

(space-before-function-paren)